src/matrix.jl

```diff
@@ -95,6 +95,21 @@ function similar(M::IntervalMatrix)
     return IntervalMatrix(similar(M.mat))
 end
 
+# constructor from a scalar matrix
+function IntervalMatrix(A::AbstractMatrix{T}) where {T<:Number}
+    return IntervalMatrix(map(Interval, A))
+end
+
+# conversion from scalar matrix to interval matrix
+function convert(::Type{<:IntervalMatrix}, A::AbstractMatrix{T}) where {T<:Number}
+    return IntervalMatrix(map(x -> convert(Interval, x), A))
+end
+
+# no-op
+function convert(::Type{<:IntervalMatrix}, A::IntervalMatrix)
+    return A
+end
```

Comment:
This is not correct for `convert(IntervalMatrix{T2}, A::IntervalMatrix{T1})` with `T1 != T2`?

Reply:
True; for the general case we should appropriate type promotion.